swift/Sources/FlatBuffers/FlatBufferBuilder.swift

```diff
@@ -510,4 +501,82 @@ extension FlatBufferBuilder: CustomDebugStringConvertible {
         { finished: \(finished), serializeDefaults: \(serializeDefaults), isNested: \(isNested) }
         """
     }
+
+    /// VTableStorage is a class to contain the VTable buffer that would be serialized into buffer
+    @usableFromInline internal class VTableStorage {
```

Comment:
Why is having this in a separate class a good idea? Could be even faster if directly in the builder?

Reply:
I think it's better to have the buffer in a class since we dont want to resize. Reading documentations for protobuf and [swiftnio bytebuffer](https://github.com/apple/swift-nio/blob/master/Sources/NIO/ByteBuffer-core.swift), they usually save the unsafememorybuffers in classes so they dereference the pointer without memory leaks since a struct have a deinit in swift

Reply:
It was just an observation, I trust you know what works better in Swift :)